### DISCUSSION TITLE
Docker Image Version

### DISCUSSION THREAD

**Discussion by mokun** [mars-sim/mars-sim#1004] [General]
**Full Changelog**: https://github.com/mars-sim/mars-sim/compare/v3.5.0

### A. Editions
Version 3.6.0 are available in 4 editions : 
1. Jar Swing Edition              : `3.6.0_swing_java17.jar`
2. Jar Console Edition         : `3.6.0_console_java17.jar`
3. Debian Swing Edition              : `3.6.0_swing_java17.deb`
4. Debian Console Edition         : `3.6.0_console_java17.deb`

In the console edition above, either a beryx console or a SSH terminal can be used as the means of interacting with settlers, robots, settlements, vehicles, etc. in mars-sim. See wiki with regard to setting up [SSH console](https://github.com/mars-sim/mars-sim/wiki/Remote-Console-Connection).

**Comment by mokun** [comment]
@bevans2000 

How do I update the docker image to the latest version ?

I look at this [page](https://github.com/mars-sim/mars-sim/pkgs/container/mars-sim%2Fmars-sim/61124093?tag=3.4.1) and it shows old versions.

Do we want to stick with the old name format with `headless` or with `console`?

**Reply by bevans2000** [reply]
I thought it was setup to be automatic from the Action. I will have a look

**Reply by mokun** [reply]
btw, I might have clicked on a dependabot's update for using Java 20 for docker image in the past.

**Reply by bevans2000** [reply]
I'm going to sort this as part of #1007. The GitHub Action to create the Docker image is not automatic but manually triggered. 
I will change this to create a Docker image when a tag is applied.

**Comment by bevans2000** [comment]
This is fixed now
